backend/tracim_backend/migration/env.py

```diff
@@ -21,6 +22,8 @@
 env_sqlalchemy_url = os.environ.get("TRACIM_SQLALCHEMY__URL")
 if env_sqlalchemy_url:
     config.set_main_option("sqlalchemy.url", env_sqlalchemy_url)
+if not config.get_main_option("sqlalchemy.url"):
+    raise ConfigurationError("SQLALCHEMY__URL is mandatory")
```

Comment:
It seems not clear to me : you check `sqlalchemy.url` but you raise `SQLALCHEMY__URL` error. Do your code check the url from both config and envvar configuration?

Reply:
Yes, it does it, sqlalchemy.url in config of alembic is at this point the correct value to check (there is no CFG here).
If you check the lines before we override config if there is env_var setted.

Reply:
as SQLALCHEMY__URL is the parameter name of tracim that the best thing to return as mandatory.